### PR TITLE
Styling & viewport settings layout

### PR DESCRIPTION
**Pull request by marijnvanderhorst**
#### Deployment instructions
Only merge after target branch is development.

#### What have I done
Styled the existing components using a dark theme. Also added the basic structure for the viewport settings component, containing just 3 tabs with mock content.

#### Testing
/

### CODE REVIEW

File: src/_variables.scss [hunk 1]
```diff
@@ -3,4 +3,7 @@ $toolbar-height: 4rem;
 /* Override Bootstrap dark theme color to match Oce's */
 $theme-colors: ( // TODO possibly style theme according to oce color palette
   "dark": #1d1d1b
-);
\ No newline at end of file
+);
+
+$body-bg: #313d4c; // Override Bootstrap background color.
+$bg-secondary: #4e5e73; // New variable for secondary background color.
```
Comment (Dacaspex) [inline src/_variables.scss]: I would not call it "new" because will it be "new" after a while?

Comment (Dacaspex) [inline src/_variables.scss]: Also, perhaps we should agree on comment styling that we never put comments on the same line? Just a thought open for discussion!

File: src/app/viewport-settings/viewport-settings.component.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<ngb-tabset justify="fill">
```
Comment (Dacaspex) [inline src/app/viewport-settings/viewport-settings.component.html]: Perhaps we can put the `viewport-settings` in the `viewport` folder? Like this:
```
viewport/
  viewport-settings/
    viewport-settings.html|ts|scss|spec
  viewport.html|ts|scss|spec
```

File: src/styles.scss [hunk 3]
```diff
@@ -2,3 +2,19 @@
 @import 'variables';
 @import '~bootstrap/scss/bootstrap';
 @import "~@ng-select/ng-select/themes/default.theme.css";
+
+/* Override Bootstrap nav tab styles to work properly with our theme.*/
+.nav-tabs .nav-link.active {
+  color: white;
+  background-color: $bg-secondary;
+  border-bottom: solid $bg-secondary;
```
Comment (Dacaspex) [inline src/styles.scss]: This is the same colour as the background, do you want to simulate padding here? Or why is it the same colour?

File: src/_variables.scss (hunk 1 above)
Comment (marijnvanderhorst) [reply src/_variables.scss]: New as in "not an overwrite like the other variables in the file", but I agree it's unclear. Will rephrase.

Comment (marijnvanderhorst) [reply src/_variables.scss]: I agree it's better for consistency and extendability of comment length

File: src/app/viewport-settings/viewport-settings.component.html (hunk 2 above)
Comment (marijnvanderhorst) [reply src/app/viewport-settings/viewport-settings.component.html]: I don't really agree, as it's really a separate component and not a sub-component of the viewport.

Comment (Dacaspex) [reply src/app/viewport-settings/viewport-settings.component.html]: Fair point, we can leave it like this

File: src/styles.scss (hunk 3 above)
Comment (marijnvanderhorst) [reply src/styles.scss]: Bootstrap's styles need to be overwritten, but I've made some improvements to the overwrite in 36039c5.

File: src/_variables.scss [hunk 4]
```diff
@@ -5,5 +5,8 @@ $theme-colors: ( // TODO possibly style theme according to oce color palette
   "dark": #1d1d1b
 );
 
-$body-bg: #313d4c; // Override Bootstrap background color.
-$bg-secondary: #4e5e73; // New variable for secondary background color.
\ No newline at end of file
+/* Override Bootstrap background color. */
```
Comment (Dacaspex) [inline src/_variables.scss]: Why `/* comment */` and not `// comment`?

Comment (marijnvanderhorst) [reply src/_variables.scss]: ¯\_(ツ)_/¯

**Pull request merged**